Octokit.Tests/Clients/MigrationsClientTests.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NSubstitute;
+using Xunit;
+
+namespace Octokit.Tests.Clients
+{
+    public class MigrationsClientTests
+    {
+        public class TheGetMethod
+        {
+            [Fact]
+            public void RequestsCorrectUrl()
+            {
+                var connection = Substitute.For<IApiConnection>();
+                var client = new MigrationsClient(connection);
+
+                client.Get("fake", 69);
+
+                connection.Received().Get<Migration>(
```

Comment:
Now you're using the Accepts Header, all the mock `Connection.Received()` checks will need to include that parameter as well (and just a reminder that in the unit test, you should hardcode the expected accept header string ie dont just use the `AcceptHeaders` helper class... otherwise if there was ever a typo in it, the unit tests would pass!  So we specify it "again" on the unit test side)